keps/2076-kueuectl/README.md

```diff
@@ -0,0 +1,342 @@
+# KEP-2076: Kueuectl
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Create ClusterQueue](#create-clusterqueue)
+  - [Create LocalQueue](#create-localqueue)
+  - [List ClusterQueue](#list-clusterqueue)
+  - [List LocalQueue](#list-localqueue)
+  - [List Workloads](#list-workloads)
+  - [Stop ClusterQueue](#stop-clusterqueue)
+  - [Resume ClusterQueue](#resume-clusterqueue)
+  - [Stop LocalQueue](#stop-localqueue)
+  - [Resume LocalQueue](#resume-localqueue)
+  - [Stop Workload](#stop-workload)
+  - [Resume Workload](#resume-workload)
+  - [Pass-through](#pass-through)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+We want to create a command line tool for Kueue that allows to:
+
+* list Kueue's objects with easy to use Kueue-specific filtering,
+* create Local and ClusterQueues without touching yamls,
+* perform management operations on LQs, CQs and Workloads.
+
+## Motivation
+
+Currently many administrative operations around Kueue are largely inconvenient. 
+They require full API understanding, are relatively error-prone or are simply
+impossible without writing a custom mini script or complex pipe processing.
+
+### Goals
+
+* Provide a command line tool for system administrator to:
+
+    * Create ClusterQueues and LocalQueues.
+    * Listing *Queues and Workloads that meet certain criteria.
+    * Stopping and resuming execution in ClusteQueuesr and LocalQueues.
+    * Stooping and resuming individual Workloads.
+    * Migrating workloads between LocalQueues.
```

Comment:
It doesn't look like this is explained below? Remove it?

Reply:
Fixed.